modules/sdk-core/src/bitgo/staking/iStakingWallet.ts

```diff
@@ -23,11 +23,22 @@ export interface DelegationRequest {
   validator: string;
 }
 
+/**
+ * Represents the options for staking.
+ * @typedef {Object} StakeOptions
+ * @property {string} [amount] - amount to stake
+ * @property {string} [clientId] - clientId
+ * @property {string} [validator] - chosen validator
+ * @property {string} [duration] - delegation duration: a numeric string, in days or cycles
+ * @property {string} [subType] - coin sepcific staking subtype
+ * @property {DelegationRequest[]} [delegationRequests] - The delegation requests
+ */
 export interface StakeOptions {
   amount?: string;
   clientId?: string;
   validator?: string;
-  durationSeconds?: string;
+  duration?: string;
+  subType?: string;
```

Comment:
Can we enumerate the different types instead of just a string? When is this expected to be populated?

Reply:
if you go over this again, please use this jsdoc syntax

```
interface I {
 /** foo */
 foo: bar
}
```